documentation/asciidoc/topics/proc_disabling_fips_mode_cr.adoc

```diff
@@ -0,0 +1,32 @@
+[id="disabling-fips-mode-CR_{context}"]
+= Disabling FIPS mode in your `Infinispan` CR
+
+The {openshiftplatform} can use certain Federal Information Processing Standards (FIPS) components that ensure {openshiftshort} clusters meet the requirements of a FIPS compliance audit. 
+This might cause issues when you want your {brandname} instance to run on any {openshiftshort} cluster that has FIPS mode enabled. 
+{brandname} {ProductVersion} does not support FIPS mode, so you must disable FIPS mode in your `Infinispan` CR.
+
+After you disable FIPS mode in your `Infinispan` CR, any mechanism that uses a JVM, such as {brandname} Server, {brandname} CLI, or Gossip router, ignores FIPS mode. 
+This happens, because the JVM no longer loads FIPS-related cryptographic libraries on implementation startup. 
+
+[NOTE]
+====
+If an {openshiftshort} cluster has FIPS mode disabled, you do not need to explicitly disable FIPS mode in your `Infinispan` CR configuration.
```

Comment:
As FIPS is disabled by default, should we switch this note to emphasise that FIPS only has to be explicitly disabled IF FIPS has been enabled?